lib/lib.es2015.symbol.wellknown.d.ts

```diff
@@ -236,7 +236,10 @@ interface String {
     /**
      * Replaces text in a string, using an object that supports replacement within a string.
      * @param searchValue A object can search for and replace matches within a string.
-     * @param replaceValue A string containing the text to replace for every successful match of searchValue in this string.
+     * @param replaceValue A string containing the text to replace for successful
```

Comment:
In general, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) is a pretty good source for wording. In this case, their documentation explains that if you pass a string, it will only replace the first match. I would start from that instead of attempting to explain regex syntax inline.

Reply:
Removed the lib change and got the wording gist from MDN.
What command do I run to see the lint errors and are they related to this?